--- a/util_spectral_scan/Makefile
+++ b/util_spectral_scan/Makefile
@@ -74,7 +74,7 @@
 
 ### Compile main program
 $(OBJDIR)/$(APP_NAME).o: src/$(APP_NAME).c | $(OBJDIR)
-	$(CC) -c $< -o $@ $(CFLAGS) -Iinc -I../libloragw/inc
+	$(CC) -c $< -o $@ $(CFLAGS) -Iinc -I../libloragw/inc -I../libtools/inc
 
 ### Link everything together
 $(APP_NAME): $(OBJDIR)/$(APP_NAME).o $(LIB_PATH)/libloragw.a
--- a/util_boot/Makefile
+++ b/util_boot/Makefile
@@ -74,7 +74,7 @@
 
 ### Compile main program
 $(OBJDIR)/$(APP_NAME).o: src/$(APP_NAME).c | $(OBJDIR)
-	$(CC) -c $< -o $@ $(CFLAGS) -Iinc -I../libloragw/inc
+	$(CC) -c $< -o $@ $(CFLAGS) -Iinc -I../libloragw/inc -I../libtools/inc
 
 ### Link everything together
 $(APP_NAME): $(OBJDIR)/$(APP_NAME).o $(LIB_PATH)/libloragw.a
--- a/util_chip_id/Makefile
+++ b/util_chip_id/Makefile
@@ -74,7 +74,7 @@
 
 ### Compile main program
 $(OBJDIR)/$(APP_NAME).o: src/$(APP_NAME).c | $(OBJDIR)
-	$(CC) -c $< -o $@ $(CFLAGS) -Iinc -I../libloragw/inc
+	$(CC) -c $< -o $@ $(CFLAGS) -Iinc -I../libloragw/inc -I../libtools/inc
 
 ### Link everything together
 $(APP_NAME): $(OBJDIR)/$(APP_NAME).o $(LIB_PATH)/libloragw.a
